src/Transpiler.ts

```diff
@@ -497,7 +497,7 @@ export abstract class LuaTranspiler {
         // Add header
         let result = "";
         for (const variableDeclaration of (node.initializer as ts.VariableDeclarationList).declarations) {
-            result += this.indent + this.transpileVariableDeclaration(variableDeclaration);
+            result += this.indent + this.transpileVariableDeclaration(variableDeclaration) + "\n";
```

Comment:
Should there also be a `;` here?

Reply:
Not really, you know a while will always follow, so there's no ambiguity. I've tried not to add ; where they weren't needed.